build_image.sh

```diff
@@ -111,7 +111,7 @@ if [ "$IMAGE_TYPE" = "onie" ]; then
     mkdir -p `dirname $OUTPUT_ONIE_IMAGE`
     sudo rm -f $OUTPUT_ONIE_IMAGE
 
-    generate_device_list "./installer/$TARGET_PLATFORM/platforms_asic"
+    generate_device_list "./installer/platforms_asic"
```

Comment:
Can you please clarify your suggestion here? 

The difference you have highlighted is simply the result of me relocating the `platforms_asic` file because there are now no longer arch-specific installer directories, there is just a common installer.